livesync/indico_livesync/handler.py

```diff
@@ -165,16 +137,14 @@ def _note_changed(note, **kwargs):
 def _attachment_changed(attachment_or_folder, **kwargs):
     folder = getattr(attachment_or_folder, 'folder', attachment_or_folder)
     if folder.link_type not in (LinkType.category, LinkType.session):
-        _register_change(folder.object, ChangeType.data_changed)
+        _register_change(folder.object.event_new, ChangeType.data_changed)
 
 
 def _apply_changes(sender, **kwargs):
     if not hasattr(g, 'livesync_changes'):
         return
     for ref, changes in g.livesync_changes.iteritems():
-        if is_ref_excluded(ref):
-            continue
-        LiveSyncQueueEntry.create(changes, ref)
+        LiveSyncQueueEntry.create(changes, ref, excluded_categories=get_excluded_categories())
```

Comment:
does it really make sense to call it inside the loop? i know it's memoized, but it still seems cleaner to do it outside